File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/key-activities-at-business-unit_item.ts


import { IKeyActivitiesAtBusinessUnit_Item } from './ikey-activities-at-business-unit_item';

export class KeyActivitiesAtBusinessUnit_Item implements IKeyActivitiesAtBusinessUnit_Item {
  
	public KeyActivityID: number = 0;
	public Code: string = "";
	public BusinessUnitID: number = 0;
	public Acronym: string = "";
	public BusinessUnitName: string = "";
	public BusinessUnitActive: boolean = false;
	public KeyActivityBusinessUnitActive: boolean = false;
  
}

File: GTTS/INL.TrainingService.Models/KeyActivitiesAtBusinessUnit_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.TrainingService.Models
{
    public class KeyActivitiesAtBusinessUnit_Item : IKeyActivitiesAtBusinessUnit_Item
    {
        public int KeyActivityID { get; set; }
        public string Code { get; set; }
        public long BusinessUnitID { get; set; }
        public string Acronym { get; set; }
        public string BusinessUnitName { get; set; }
        public bool BusinessUnitActive { get; set; }
        public bool KeyActivityBusinessUnitActive { get; set; }
    }
}
